rubrik_cdm/data_management.py

```diff
@@ -990,3 +990,109 @@ def sql_live_mount(self, db_name, date, time, sql_instance=None, sql_host=None,
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            dict -- The full response of `DELETE '/vmware/vm/snapshot/mount/{id}?force={bool}'.
+        """
+
+
+        self.log("vsphere_live_unmount: Searching the Rubrik cluster for the Live Mount vSphere VM '{}'.".format(mounted_vm_name))
+        mounted_vm_id = self.object_id(mounted_vm_name, 'vmware', timeout=timeout)
+
+        self.log("vsphere_live_unmount: Getting the vSphere VM mount information from the Rubrik cluster.")
+        mount_summary = self.get('v1', '/vmware/vm/snapshot/mount', timeout=timeout)
+
+        self.log("vsphere_live_unmount: Getting the mount ID of the vSPhere VM '{}'.".format(mounted_vm_name))
+        for mountedvm in mount_summary['data']:
+            if mountedvm['mountedVmId'] == mounted_vm_id:
+                mount_id = mountedvm['id']
+                break
+        else:
+            raise InvalidParameterException("The mounted VM '{}' does not exist, please provide a valid instance".format(mounted_vm_name))
+
+        try:
+            mount_id
+        except NameError:
+            raise InvalidParameterException("The mounted vSphere VM '{}' does exist, please check the name you provided.".format(
+                mounted_vm_name))
+        else:
+            #config = {}
+            #config['force'] = force
+
+            self.log(
+                "vsphere_live_mount: Unmounting the vSphere VM '{}'.".format(mounted_vm_name))
+
+            return self.delete('v1', '/vmware/vm/snapshot/mount/{}?force={}'.format(mount_id, force), timeout)
+
+    def sql_live_unmount(self, mounted_db_name, sql_instance=None, sql_host=None, force=False, timeout=30):  # pylint: ignore
+        """Delete a Microsoft SQL Live Mount from the Rubrik cluster. 
+
+        Arguments:
+            mounted_db_name {str} -- The name of the Live Mounted database to be unmounted. 
+
+        Keyword Arguments:
+            sql_instance {str} -- The name of the MSSQL instance managing the Live Mounted database to be unmounted. 
+            sql_host {str} -- The name of the MSSQL host running the Live Mounted database to be unmounted. 
+            force {bool} -- Remove all data within the Rubrik cluster related to the Live Mount, even if the SQL Server database cannot be contacted. (default: {False})
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            dict -- The full response of `DELETE /mssql/db/mount/{id}?force={bool}`.
+        """
+
+
+        if sql_instance is None or sql_host is None:
+                raise InvalidParameterException(
+                    "To live mount a mssql database the 'sql_instance' and 'sql_host' paramaters must be provided.")
```

Comment:
`"To live mount a mssql database the 'sql_instance' and 'sql_host' paramaters must be provided."`

`live mount` should be something related to unmount